INSTALL

```diff
@@ -0,0 +1,168 @@
+           Netatalk Installation
+           =====================
+
+Quick start
+===========
+
+Netatalk can be built by using Meson or GNU Autotools.
+The meson build system is the recommended build system for the main
+branch, autotools support may be be removed at some point in the future.
+
+Meson only supports out-of-tree builds, and must be passed a directory to put
+built and generated sources into. We'll call that directory "build" here. It's
+recommended to create a separate build directory for each configuration you
+might want to use.
+
+Configuration
+==============
+
+Basic configuration is done with:
+
+cd netatalk
+meson setup build
+
+
+This will create the build directory. If any dependencies are missing, you can
+install them, or try to remove the dependency with a Meson configuration option
+(see below).
+
+
+Configuration flags
+===================
+
+When using Meson, to review the options which Meson chose, run:
+
+meson configure build
+
+
+With additional arguments meson configure can be used to change options for a
+previously configured build directory. All options passed to this command are in
+the form `-D"option"="value"`. For example:
+
+meson setup build -Dprefix=/tmp/install
+
+See `meson_options.txt` for details of netatalk-specific options, and
+https://mesonbuild.com/Builtin-options.html for details of generic
+Meson options.
+
+Building
+========
+
+To build with meson, here is a quick guide:
+
+To compile:
+ninja -C build
+
+To install (Netatalk requires root privileges on install):
+sudo ninja -C build install
+
+To test (requires the -Dbuild-tests=true flag at configure time):
+cd build && meson test
+
+Full build instructions can be found on mesonbuild website:
+https://mesonbuild.com/Running-Meson.html
+
+External software dependencies
+==============================
+
+Required:
+
+  - libevent 2
+  - libgcrypt version 1.2.3 or later (for DHX2 UAM support)
+  - Berkeley DB version 4.6.0 or later
+  - perl
+
+Optional:
+
+  - avahi or mDNSresponder  (for Zeroconf support)
+  - cracklib                (for cracklib support)
+  - cups                    (for papd support)
+  - Docbook XSL stylesheets (for manpages & manual documentation)
+  - GLib 2 and dbus-glib    (for afpstats support)
+  - Kerberos V              (for krbV UAM support)
+  - libressl or OpenSSL@1.1 (for Randnum and DHX UAM support)
+  - tcpwrap                 (for TCP wrapper support)
+  - xsltproc                (for manpages and mananual documentation)
```

Comment:
Maybe list this together with “docbook” above? You need both to achieve anything. 

BTW, I found recently that you can run `dblatex` on manual.xml to get a nicely formatted PDF manual via the latex formatter.